ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -79,6 +94,108 @@
         }
     }
 
-    Utils.initializeEditDialog(EDIT_DIALOG)(handleAsyncSubmissionAndThankYouOption, handleSubmitAction);
+    function loadSubmitActionSubDialogClientLibs(dialog) {
+        var containerSubmitAction = dialog.find(CONTAINER_SUBMITACTION)[0];
+        containerSubmitAction.on("change", function(){
+            loadRestEndPointDialogClientlibs(dialog);
+            loadFDMDialogClientlibs(dialog);
+            loadEmailDialogClientlibs(dialog);
+        });
+    }
+
+    function showDorBindRefSelector (dialog) {
+        var enableDORSubmissionCheckBoxElement = dialog.find(FDM_ENABLE_DOR_CHECK_BOX)[0],
+            $dorBindRefSelector = Utils.selectElement("input", './bindRef'),
+            parentBindRefSelector = "span.bindRefSelector";
+        // ensuring the event is triggered only once, since the widget is going into inconsistent state
+        // due to multiple event firing
+        if (!$(enableDORSubmissionCheckBoxElement).data("eventTriggered")) {
+            // trigger the content loaded event for bind ref to initialize itself
+            $(document).trigger("fdBindRef-contentLoaded");
```

Comment:
this is internal event. -1 to exposing it here.

Reply:
will remove and write custom function for the behaviour.

Reply:
removed